tests/sles4sap/hana_cluster.pm

```diff
@@ -97,4 +97,11 @@ sub run {
     check_cluster_state;
 }
 
+sub post_fail_hook {
```

Comment:
Function `post_fail_hook` already exists in `lib/sles4sap`, so Travis is failing. Please fix this.